lib/model/CEventRateModel.cc

```diff
@@ -422,6 +422,13 @@ bool CEventRateModel::computeProbability(std::size_t pid,
             continue;
         }
 
+        if (this->initialCountWeight(
```

Comment:
I think the intention would be much clearer if we had a new method called something `shouldSkipUpdate`. This could be implemented on `CModel` and behind the scene just do 
```
return this->initialCountWeight(
                feature, pid, model_t::INDIVIDUAL_ANALYSIS_ATTRIBUTE_ID,
                model_t::sampleTime(feature, startTime, bucketLength)) != 1.0
```

Reply:
I also wonder whether it is strictly more correct to do this before checking `shouldIgnoreResult` and continuing because we probably want to not update quantiles even if we are ignoring the result.